public/app/features/dashboard-scene/pages/DashboardScenePage.tsx

```diff
@@ -86,7 +86,7 @@ export function DashboardScenePage({ match, route, queryParams, history }: Props
   }
 
   return (
-    <UrlSyncContextProvider scene={dashboard}>
+    <UrlSyncContextProvider scene={dashboard} updateUrlOnInit={true} createBrowserHistorySteps={true}>
```

Comment:
I think we need to add this in the Public Dashboard component too: https://github.com/grafana/grafana/blob/d8b6a5bde2737dadbf730f3ccd31d275c94e9968/public/app/features/dashboard-scene/pages/PublicDashboardScenePage.tsx#L64

It make sense this behavior for the public dashboard